<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>c_componentElemView</class>
 <widget class="QWidget" name="c_componentElemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="quantityUnitLabel">
      <property name="geometry">
       <rect>
        <x>-60</x>
        <y>40</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ingredientLineEdit">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>40</y>
        <width>108</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="unitComboBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>72</width>
        <height>22</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>-80</x>
        <y>43</y>
        <width>16</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>417</x>
        <y>40</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="quantitySpinBox">
      <property name="geometry">
       <rect>
        <x>-7</x>
        <y>40</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
